MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Media/MediaImageFormView.js

```diff
@@ -113,6 +117,9 @@ class MediaImageFormView extends MediaFormView
                 viewContext._cropParam['boundy'] = bounds[1];
                 viewContext._cropParam['jcrop_api'] = this;
                 viewContext._cropParam['$preview'].appendTo(this.ui.holder);
+                viewContext._jcrop_api = this;
+                this.viewContext = viewContext;
```

Comment:
this line is useless no ?

Reply:
no, this.viewContext is used on line 179, 180, 181 and 182